basics/data-import/pinot-file-system/README.md

```diff
@@ -25,33 +25,31 @@ Pinot lets you choose a distributed file system provider. The following file sys
 
 ## Enabling a File System
```

Comment:
```suggestion
## Enabling a file system
```

Reply:
global search for:
File System/File system/Filesystem/filesystem >> file system